frontend/src/components/QueueTableDropdown.js

```diff
@@ -1,12 +1,23 @@
-import React, { useState } from "react"
+import React, { useContext } from "react"
 import PropTypes, { oneOfType, string, number } from "prop-types"
 import MenuItem from "@material-ui/core/MenuItem"
 import Select from "@material-ui/core/Select"
+import { QueueStoreContext } from "../stores/QueueStore"
 
 const QueueTableDropdown = props => {
-  const [value, setValue] = useState(props.initialValue)
```

Comment:
The original useState is correct since value should reset on each re-render if you don't.

Reply:
Here I have to disagree.

I had it as you suggested, but it doesn't work correctly.  If you persist state in the component with useState, then if you change a status--e.g. to "Stepped Out" in one queue--and then change tabs, you will find it's also changed in the other tabs.  

Instead you want the state persisted in the backend, and the data is refreshed from the backend when you change tabs. This is why I set value=props.initialValue

Reply:
I'll need to play with it to get it. I probably don't understand the current context of this component.